app/src/main/java/com/fidesmo/sec/servicedeliverydemo/ServiceDeliveryActivity.java

```diff
@@ -87,7 +88,7 @@ void callFidesmoApp(String action, String spId, String serviceId) {
         try {
             // create Intent to one of the two Actions exposed by the Fidesmo
             // App
-            Intent intent = new Intent(action, Uri.parse(SERVICE_URI + spId + "/" + serviceId));
+            Intent intent = new Intent(action, Uri.parse(SERVICE_URI + spId + "/services/" + serviceId));
```

Comment:
Then we need to update devportal documentation. I'm opening an issue for you...
 [this issue](https://github.com/fidesmo/devportal/issues/103)